app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <h2>Sign up</h2>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
```

Comment:
- [ ] Great job working on the sign up page but please as shown in the guidelines your design should be the same as the given design so kindly make the layout the same including the colors as mentioned in the reqiorements ⭐🚀🔥

![image](https://github.com/Timbar09/Budget-App/assets/58553711/5c51d414-39a8-4dfa-a1ed-ee08e3455b00)


## please change the color as well 🔥💪🏻💪🏻
- Your
![image](https://github.com/Timbar09/Budget-App/assets/58553711/297ed493-8eb5-4c8d-b0b6-9d5022c37c20)

- Given
![image](https://github.com/Timbar09/Budget-App/assets/58553711/d15a2a99-6b6c-4897-b271-57ee722339b4)